gui/public/theme.css

```diff
@@ -59,7 +59,7 @@ h1 {
 /* Avoid awful default scrollbars in most OS/Browsers, especially in horizontally scrolling text displays. */
```

Comment:
I missed this in an earlier change, but **thank you** for doing custom scrollbars!